lib/passenger.rb

```diff
@@ -20,5 +20,14 @@ def get_drivers
     def add_trip(trip)
       @trips << trip
     end
+
+    def total_spent
+      @trips.map{ |trip| trip.cost }.inject(0, :+).round(2)
```

Comment:
Use consistent whitespace. This feels cramped to me:

```ruby
@trips.map{ |trip| trip.cost }
```

Idiomatic Ruby looks more like:

```ruby
#           v    inserting a space here
#                            v    the space on the other side is balanced
@trips.map { |trip| trip.cost }
```